BaseApp/COD50004.txt

```diff
@@ -0,0 +1,50 @@
+OBJECT Codeunit 50004 DotNet_XmlReader

```

Comment:
We do have a native implementation of this in the al language. Is there any reason why you need this impementation instread? We would have to make the function here [Internal], as we don't want two competing constructs for the same purpose.